heim-runtime/src/lib.rs

```diff
@@ -26,6 +26,7 @@ use std::future::Future;
 pub use futures::pin_mut as pin;
 
 pub mod fs;
+pub mod linux;
```

Comment:
```suggestion
#[cfg(target_os = "linux")]
pub mod linux;
```

Why don't we can make the whole mod enabled conditionally?

Reply:
Oh yeah that's much better!